content/spin/serverless-ai-tutorial.md

```diff
@@ -232,6 +232,14 @@ Now let's use the Spin SDK to access the model from our app:
 
 {{ startTab "Rust"}}
 
+> Note: The following Rust source code, for this sentiment analysis example, uses serde. Please add the following dependencies to your Rust application's `Cargo.toml` file before you build/run the application.
+
+```toml
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0.85"
+```
+The following Rust source code is an example of what goes in your `src/lib.rs` file:
```

Comment:
I wouldn't describe this as an "example" - this is what we want them to write isn't it?

```suggestion
Change your `lib.rs` file to read as follows:
```

(or better phrasing, I'm not mad keen on 'read' here)